fourhills/exceptions.py

```diff
@@ -1,5 +1,5 @@
 
-class FourhillsError(RuntimeError):
+class FourhillsError(Exception):
```

Comment:
What's the purpose of this change? RuntimeError is a less generic thing to inherit from than Exception